bittensor/dendrite.py

```diff
@@ -164,7 +164,10 @@ async def aclose_session(self):
         """
         if self._session:
             await self._session.close()
-            self._session = None
+            try:
+                self._session = None
```

Comment:
Mind explaining this code block a bit? Would assigning session as `None` raise an exception?

Reply:
Sure! It has errored for me in an edge case but I haven't been able to reproduce since.  May be more conservative than necessary. We can hold off on this if no one is having noticeable issues.